src/mbgl/text/glyph_set.cpp

```diff
@@ -143,7 +153,8 @@ void GlyphSet::lineWrap(Shaping &shaping, const float lineHeight, const float ma
                 || shape.glyph == 0xb7 /* middle dot */
                 || shape.glyph == 0x200b /* zero-width space */
                 || shape.glyph == 0x2010 /* hyphen */
-                || shape.glyph == 0x2013 /* en dash */) {
+                || shape.glyph == 0x2013 /* en dash */
```

Comment:
We could move all of those as well to a sister function of `allowsIdeographicBreaking` in the `i18n` namespace